polyfill/Blob.js

```diff
@@ -2,12 +2,13 @@
 // Use of this source code is governed by a MIT-style license that can be
 // found in the LICENSE file.
 
-import RNFetchBlob from '../index.js'
+import {NativeModules} from 'react-native';
```

Comment:
Consider having module in `polyfill/RNFetchBlob.js`:

```javascript
import { NativeModules } from 'react-native';

const RNFetchBlob = NativeModules.RNFetchBlob

export default RNFetchBlob;
```

Seems like trivial module but allows to have single shared source.